code/game/machinery/hologram.dm

```diff
@@ -90,6 +97,14 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 				return
 			if(last_request + 200 < world.time) //don't spam other people with requests either, you jerk!
 				last_request = world.time
+				var/obj/item/card/id/I = user.GetIdCard()
+				if(!user.GetIdCard())

```

Comment:
You already assigned `user.GetIdCard()` to `I`, so you can just check for `if(!I)`

Reply:
Done